platform/feature-api/communication/postoffice/README.md

```diff
@@ -49,7 +49,7 @@ const simpleEndpointImpl: ServerOf<SimpleEndpoint> = {
         throw new Error(`${parama}, ${paramb.join()}`)
 };
 
-server(serverPort, endpointServer(simpleEndpointImpl));
+server(serverPort, backendServer(simpleBackendImpl));
```

Comment:
Then here "backend" becomes a bit iterative... This is not the point where it's defined, let's get a bit further and see if we can come up with some alternatives.